src/main/kotlin/com/terraformation/backend/documentproducer/model/Variable.kt

```diff
@@ -441,15 +416,22 @@ data class SectionVariable(
         throw VariableValueInvalidException(id, "Display style is required for variable injections")
       }
 
-      val usedVariableId = value.value.usedVariableId
+      // TODO validate this somewhere else
```

Comment:
What was wrong with the existing validation setup?

Reply:
I had removed this when I thought I needed to split up the fetch contexts, I put it all back and everything is still working.